doc/output-details-and-examples/src/output-files/eplustbl-lt-ext-gt.tex

```diff
@@ -1279,7 +1279,7 @@ \subsection{HVAC Sizing Summary}\label{hvac-sizing-summary}
 
 \emph{Zone Name(s)} This is the zone, or list of zones, found to be associated with this coil.  The zone name is defined by the user in the Zone object.  For multizone systems, the coils can have a list of zone names with the individual names separated by a semicolon. The zones listed here are those used to generate (aggregate and volume-weighted average) values for the room conditions and loads. Text string. 
 
-\emph{System Sizing Method Concurrence} This field applies to ``AirLoop'' coils.  It describes if the central air system sizing was done using ``Coincident'' or ``Non-Coincident'' method. This refers to how the ideal loads sizing routines process the time sequence of zone loads.  The concurrent sum of peaks, or coincident or "block", is often lower than the sum of the individual peaks regardless of when they occur, or non-conicident sum.  This field is not applicable to ``Zone Equipment'' coils and will show ``N/A'' here.  This corresponds to the input field called Type of Zone Sum to Use in the Sizing:System input object. Text string.
+\emph{System Sizing Method Concurrence} This field applies to ``AirLoop'' coils.  It describes if the central air system sizing was done using ``Coincident'' or ``Non-Coincident'' method. This refers to how the ideal loads sizing routines process the time sequence of zone loads.  The concurrent sum of peaks, or coincident or "block", is often lower than the sum of the individual peaks regardless of when they occur, or non-coincident sum.  This field is not applicable to ``Zone Equipment'' coils and will show ``N/A'' here.  This corresponds to the input field called Type of Zone Sum to Use in the Sizing:System input object. Text string.
```

Comment:
This line has a quotation mark character that should be converted.  We should most likely do an intelligent search/replace in the code to clean these up.  If you have changes to make for this PR, you might go ahead and clean this up, but it's not a big deal.

Reply:
Ok. I will change this one.